### ISSUE TITLE
<button> tag is semantically wrong element for a11y

### ISSUE THREAD

**Issue by jackahl** [kitconcept/volto-button-block#15]
We should replace the <Button> element at https://github.com/kitconcept/volto-button-block/blob/2c32645229deb5fe7cc9d24756dc41cec21ff930/src/components/View.jsx#L38 or set a `as="span"` prop because semantically this block is just a link and not a html button. Having buttons within links is very confusing when navigating the page via keyboard and definitely a a11y violation.
